src/components/code-line.component.js

```diff
@@ -84,62 +99,45 @@ export class CodeLine extends Component {
     };
 
     return (
-      <View style={styles.container}>
-        <View style={styles.wrapper}>
-          <View
-            style={[
-              styles.lineNumbers,
-              newChunk && styles.newChunkLineNumbers,
-              change.type === 'add' && styles.addLineNumbers,
-              change.type === 'del' && styles.delLineNumbers,
-            ]}
-          >
-            <Text style={styles.codeLineNumber}>
+      <Container>
+        <Wrapper>
+          <LineNumbers newChunk change>
+            <CodeLineNumber>
               {change.type === 'del'
                 ? change.ln
                 : change.type === 'normal'
                   ? change.ln1
                   : change.type === 'add' ? '' : '...'}
-            </Text>
-            <Text style={styles.codeLineNumber}>
+            </CodeLineNumber>
+            <CodeLineNumber>
               {change.type === 'add'
                 ? change.ln
                 : change.type === 'normal'
                   ? change.ln2
                   : change.type === 'del' ? '' : '...'}
-            </Text>
-          </View>
-
-          <View
-            style={[
-              styles.codeLineContainer,
-              newChunk && styles.newChunkLineContainer,
-              change.type === 'add' && styles.addLine,
-              change.type === 'del' && styles.delLine,
-            ]}
-          >
+            </CodeLineNumber>
+          </LineNumbers>
+
+          <CodeLineContainer newChunk change>
             {(newChunk || !this.isKnownType(language)) && (
-              <Text style={[styles.codeLine, newChunk && styles.newChunkLine]}>
-                {change.content}
-              </Text>
+              <CodeLineStyled newChunk>{change.content}</CodeLineStyled>
             )}
 
             {this.isKnownType(language) && (
-              <SyntaxHighlighter
+              <SyntaxHighlighterStyled
                 language={language}
                 style={GithubStyle}
                 CodeTag={Text}
```

Comment:
You're going to need to keep Text imported for this line.

Reply:
Fixed